pixel-saver@deadalnix.me/po/de.po

```diff
@@ -0,0 +1,31 @@
+# Pixel Saver Localization
```

Comment:
I have no understanding of german, so I'm just going to trust you here. In the future, you should reserve these for later PR, to make the whole thing more digestable.